.github/CODEOWNERS

```diff
@@ -129,7 +129,6 @@ apps/theming-designer @microsoft/fluentui-react
 apps/ssr-tests-v9 @microsoft/fluentui-react-build
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31647&commitId=a1ca8291b90d83929fae30bea1a3233c5f01c0a6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Added|
|Avatar Converged.basic.normal.chromium.png|0| Added|
|Avatar Converged.size+image+badge.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>